app/templates/main/formSearch.html

```diff
@@ -48,17 +49,33 @@ <h1>{{title}}</h1>
           <div class="form-group">
             <label for="departmentSelect">Department</label>
             <div><select class="selectpicker"
-                    name='department'
-                    id='departmentSelect'
-                    data-live-search='true'
-                    title="All departments">
-                    <option value="">All departments</option>
-                  {% for department in departments %}
-                    <option value="{{department.departmentID}}"
-                            data-content="{{department.DEPT_NAME}}<small class='text-muted'> ({{department.ORG}}-{{department.ACCOUNT}})</small>">
-                      {{department.DEPT_NAME}}
-                    </option>
-                  {% endfor %}
+                    id="departmentSelect"
+                    onchange="populateTable()"
+                    title="Department">
+              {% for department in departments %}
+                  <!-- We want to either select the first option, or the option that matches the department -->
+                  {% if currentUser.isLaborAdmin or currentUser.isFinancialAidAdmin or currentUser.isSaasAdmin %}
+                      {% if loop.index == 1 or department.DEPT_NAME == department %}
+                        <option value="{{department.DEPT_NAME}}"
+                                selected>{{department.DEPT_NAME}}
+                        </option>
+                      {% else %}
+                        <option value="{{department.DEPT_NAME}}">
+                                      {{department.DEPT_NAME}}
+                        </option>
```

Comment:
instead of repeating the whole `<option>` element, just have the `if` around "selected"